test/puppetlabs/puppetdb/query/reports_test.clj

```diff
@@ -2,13 +2,22 @@
   (:require [puppetlabs.puppetdb.query :as query]
             [puppetlabs.puppetdb.examples.reports :refer [reports]]
             [clojure.test :refer :all]
+            [puppetlabs.puppetdb.cheshire :as json]
+            [clojure.walk :refer [keywordize-keys]]
+            [puppetlabs.puppetdb.query.reports :as r]
             [puppetlabs.puppetdb.fixtures :refer :all]
             [puppetlabs.puppetdb.testutils.reports :refer :all]
             [puppetlabs.puppetdb.time :refer [to-secs]]
+            [clj-time.coerce :refer  [to-string]]
             [clj-time.core :refer [now ago days]]))
 
 (use-fixtures :each with-test-db)
 
+(defn reports-equal
+  [r1 r2]
+  (let [norm (fn [x] (sort-by :hash (map #(update-in % [:resource-events] munge-resource-events) x)))]
```

Comment:
Have you seen this fail? I think it will be opaque in that it will just say reports-equal is false. To diagnose the failure you'll have to go in and change the test to give more information (i.e. println here or something). I think if you take your anon-function above, give it a name and then use it directly below like:

```
(is (= (order-events expected) (order-events actual)))
```

You'll see what failed